omod/src/main/webapp/resources/htmlforms/htn_dm_visit.xml

```diff
@@ -364,7 +364,7 @@
                     <table>
                         <tr>
                             <th>HbA1C</th>
-                            <td><obsreference conceptId="$hbA1c" showUnits="true" /></td>
+                            <td id="hba1c"><obsreference conceptId="$hbA1c" showUnits="true" /></td>
```

Comment:
This is what I did, too.

For other validation fields, the id is with the obsreference, but in this case the id is replaced with W20 (I assume a form entry module overrides the specified ID, because it adds other useful fields such as absolute minimum value). 

Just making sure you see this, in case there is a way of adding the id to the obsreference.

Reply:
Thanks @pochedls ! I moved the id within the obsreference tag.